dotcom-rendering/src/lib/useOphan.ts

```diff
@@ -1,14 +1,17 @@
 import { useEffect, useState } from 'react';
 import { getOphan } from '../client/ophan/ophan';
+import type { RenderingTarget } from '../types/renderingTarget';
 
 type Ophan = Awaited<ReturnType<typeof getOphan>>;
 
-export const useOphan = (): Ophan | undefined => {
+export const useOphan = (
+	renderingTarget: RenderingTarget,
+): Ophan | undefined => {
```

Comment:
Could we simply get the config from inside the custom hook?

If it’s available in the context, it should work.

Reply:
`useOphan` is unused and will be removed by: https://github.com/guardian/dotcom-rendering/pull/9243